src/Regridder.jl

```diff
@@ -1,21 +1,19 @@
 """
     Regridder
 
-This module contains functions to regrid information between spaces.
+This module contains functions to regrid information between CC.Spaces.
 Many of the functions used in this module call TempestRemap functions
 via ClimaCoreTempestRemap wrappers.
 """
 module Regridder
 
-using ..Utilities
-using ..TimeManager
-using ..Interfacer: CoupledSimulation, float_type, SurfaceModelSimulation, get_field, update_field!
-using ClimaCore: Meshes, Domains, Topologies, Spaces, Fields, InputOutput
-using ClimaComms
-using NCDatasets
-using ClimaCoreTempestRemap
-using Dates
-using JLD2
+import Dates
+import JLD2
+import NCDatasets as NCD
```

Comment:
NCDatasets is not a clima module. Are we happy with abbreviating here?

Reply:
good point!